docs/Tutorials/OpenAPI.md

```diff
@@ -488,18 +673,226 @@ As JSON, this could look as follows:
+
+```json
+{
+  "allOf": [
+    {
+      "$ref": "#/components/schemas/Address"
+    },
+    {
+      "$ref": "#/components/schemas/User"
+    }
+  ]
+}
+```
+
+
+## OpenApi Documentation pages
 
 If you're not using a custom OpenAPI viewer, then you can use one of the inbuilt ones with Pode - either Swagger and ReDoc, or both!
 
 For both you can customise the path to access the page on, but by default Swagger is at `/swagger` and ReDoc is at `/redoc`. If you've written your own custom OpenAPI definition then you can also set a custom path to fetch the definition.
 
-To enable either you can use the [`Enable-PodeOpenApiViewer`](../../Functions/OpenApi/Enable-PodeOpenApiViewer) function:
+To enable either you can use the [`Enable-PodeOAViewer`](../../Functions/OpenApi/Enable-PodeOAViewer) function:
 
 ```powershell
 # for swagger at "/docs/swagger"
 Enable-PodeOpenApiViewer -Type Swagger -Path '/docs/swagger' -DarkMode
 
-# or ReDoc at the default "/redoc"
+# and ReDoc at the default "/redoc"
 Enable-PodeOpenApiViewer -Type ReDoc
+
+# and RapiDoc at "/docs/rapidoc"
+Enable-PodeOAViewer -Type RapiDoc -Path '/docs/rapidoc' -DarkMode
+
+# and StopLight at "/docs/stoplight"
+Enable-PodeOAViewer -Type StopLight -Path '/docs/stoplight'
+
+# and Explorer at "/docs/explorer"
+Enable-PodeOAViewer -Type Explorer -Path '/docs/explorer'
+
+# and RapiPdf at "/docs/rapipdf"
+Enable-PodeOAViewer -Type RapiPdf -Path '/docs/rapipdf'
+
+# plus a bookmark page with the link to all documentation
+Enable-PodeOAViewer -Type Bookmarks -Path '/docs'
+```
+## Multiple OpenAPI definition
+Is possible to have multiple OpenAPI definitions inside the same Server instance.
+This feature can be very usefull in situatione like:
+- Multiple versions of the OpenAPI specification for different use case
+- Same OpenAPI but with one using the OpenAPI spec 3.0.3 and another using 3.1.0
+- Different APIs based on the IP or URL
+
+### How to use it
+Any Pode funtion that interact with OpenAPI as a `-DefinitionTag` [string[]] parameter. Here you can be specify what definition you are going to interact.
+
+Note These function accept a simple string not an array
+- `Get-PodeOADefinition`
+- `Enable-PodeOpenApi`
+- `Enable-PodeOAViewer`
+- `Add-PodeOAInfo`
+- `Test-PodeOAJsonSchemaCompliance`
+
+A new OpenAPI definition has to be created using the `Enable-PodeOpenApi` function
+
+```powershell
+Enable-PodeOpenApi -Path '/docs/openapi/v3.0' -OpenApiVersion '3.0.3'  -DefinitionTag 'v3'
+Enable-PodeOpenApi -Path '/docs/openapi/v3.1' -OpenApiVersion '3.1.0' -DefinitionTag 'v3.1'
+Enable-PodeOpenApi -Path '/docs/openapi/admin' -OpenApiVersion '3.1.0' -DefinitionTag 'admin'
+```
+
+We have also a new `Select-PodeOADefinition` that simplify the definition of multiple OpenAPI specs.
+Here an example:
+```powershell
+Select-PodeOADefinition -Tag 'v3', 'v3.1'   -ScriptBlock { #your definition
+}
+
+Select-PodeOADefinition -Tag 'admin'   -ScriptBlock { #your admin definition
+}
+```
```

Comment:
Tweak to something like follows:

```markdown
There is also [`Select-PodeOADefinition`], which simplifies the selection of which OpenAPI definition to use as a wrapper around multiple OpenAPI functions, or Route functions. Meaning you don't have to specify `-DefinitionTag` on embedded OpenAPI/Route calls:

```powershell
Select-PodeOADefinition -Tag 'v3', 'v3.1' -Scriptblock {
    Add-PodeRouteGroup -Path '/api/v5' -Routes {
        Add-PodeRoute -Method Get -Path '/petbyRef/:petId' -ScriptBlock {
            Write-PodeJsonResponse -Value 'done' -StatusCode 2005
        }
    }
}

Select-PodeOADefinition -Tag 'admin' -ScriptBlock {
    # your admin definition
}
```
```